lms/static/scripts/frontend_apps/components/ErrorDialogApp.js

```diff
@@ -0,0 +1,65 @@
+import { Fragment, createElement } from 'preact';
+import { useContext } from 'preact/hooks';
+
+import { Config } from '../config';
+
+import ErrorDisplay from './ErrorDisplay';
+import Dialog from './Dialog';
+
+/**
+ * Error dialog displayed when authorization with a third-party API via OAuth
```

Comment:
This description is still the one for `OAuth2RedirectErrorApp`. Maybe something like "A general-purpose error dialog displayed when a frontend application cannot be launched." Also worth noting in the comments that there are more specific error dialogs for some use cases (eg. `OAuth2RedirectErrorApp`).